apps/condo/domains/news/schema/NewsItemSharing.js

```diff
@@ -2,12 +2,38 @@
  * Generated by `createschema news.NewsItemSharing 'b2bApp:Relationship:B2BApp:CASCADE; newsItem:Relationship:NewsItem:CASCADE; sharingParams:Json; status:Select:processing,published,moderation,declined,archive; statusMessage:Text; lastGetRecipientsRequest:Json; lastPostRequest:Json; lastGetStatusRequest:Json; publicationViewsCount:Integer;'`
  */
 
+const dayjs = require('dayjs')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
 const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
 const { GQLListSchema, getById } = require('@open-condo/keystone/schema')
 
 const access = require('@condo/domains/news/access/NewsItemSharing')
+const {
+    BAD_STATUS_TRANSITION,
+    NOT_NEWS_SHARING_CONTEXT,
+} = require('@condo/domains/news/constants/errors')
 const { STATUSES, ALLOWED_TRANSITIONS } = require('@condo/domains/news/constants/newsItemSharingStatuses')
-const publishSharedNewsItem = require('@condo/domains/news/tasks/publishSharedNewsItem')
+const publishNewsItemSharing = require('@condo/domains/news/tasks/publishNewsItemSharing')
+
+
+
+
+const ERRORS = {
+    B2B_APP_CONTEXT_DOES_NOT_SUPPORT_NEWS_SHARING: {
+        code: BAD_USER_INPUT,
+        type: NOT_NEWS_SHARING_CONTEXT,
+        message: 'Provided b2bAppContext does not support news sharing',
+        mutation: 'createNewsItemSharing',
+        variable: ['data', 'validBefore'],
```

Comment:
There is no such field in this schema))
```suggestion
```

Reply:
![](https://i.pinimg.com/736x/4c/83/08/4c8308f44363196ff2eda2b5a26885a4.jpg)
Me